src/PluginsManager/Microsoft.Performance.Toolkit.Plugins.Core.NuGet/NuGetPluginDiscoverer.cs

```diff
@@ -87,7 +86,7 @@ public sealed class NuGetPluginDiscoverer : IPluginDiscoverer
             PackageSearchResource searchResource = await this.repository.GetResourceAsync<PackageSearchResource>(cancellationToken);
 
             var result = new Dictionary<string, AvailablePluginInfo>();
-            IEnumerable<IPackageSearchMetadata> packages = await searchResource.SearchAsync(null, new SearchFilter(false), 0, PageCount, this.logger, cancellationToken);
+            IEnumerable<IPackageSearchMetadata> packages = await searchResource.SearchAsync(null, new SearchFilter(false), 0, PageCount, null, cancellationToken);
```

Comment:
Instead of passing null, I would either
1. Remove the parameter entirely
2. Pass some kind of "null logger"
3. Pass `Logger.Create(typeof(thing))` (preferable if the parameter is needed)